core/new-gui/src/app/dashboard/component/feature-container/user-project-list/user-project-list.component.scss

```diff
@@ -17,7 +17,10 @@ button {
   border: none;
 }
 
-.edit-name-icon:hover,
```

Comment:
why are we removing this?

Reply:
@Yicong-Huang Thanks for catching this!  I did not intend to remove it, I accidentally deleted it after removing some test code.  I double checked everything and added it back.